codespell_lib/data/dictionary.txt

```diff
@@ -29528,9 +29528,6 @@ Pucini->Puccini
 Puertorrican->Puerto Rican
 Puertorricans->Puerto Ricans
 pulisher->publisher
-pullrequenst->pull requests, pull request,
```

Comment:
Moving  `pullrequest` is OK, but not `pullrequenst`.

Reply:
if I misspelled `pullRequest` as `pullRequenst` in a GraphQL query, I wouldn't expect `pull Request` as the correction though 🤔 

it also looks like there isn't much evidence for `pullrequenst` ever occurring in the wild https://github.com/search?q=pullrequenst&type=code -- so maybe I could just drop it entirely?

Reply:
True, let's keep it then.

If it has been added, that's probably because it has seen somewhere in the wild.

Reply:
I was able to find a few "pull requenst" occurrences using Google.